### ISSUE TITLE
Update screenshot in docs with retention period details

### ISSUE THREAD

**Issue by pranay01** [SigNoz/signoz.io#599]
- Should show upgraded page which has logs retention setting UI also
- Should show how the retention settings UI looks when s3 storage is enabled

**Comment by Calm-Rock** [comment]
@pranay01 have we added this somewhere in the docs ?
If not, should we include this ? And where should this be included in the docs ?

**Comment by pranay01** [comment]
I don't think it is. We also need to add details on how it works for SigNoz Cloud.

We could actually introduce a new page in docs for this titled - `Hot / Cold storage` or `long term logs storage`. This question is asked by many customers as well
